api/swagger.yaml

```diff
@@ -9064,7 +9064,7 @@ paths:
       parameters:
         - name: "fromImage"
           in: "query"
-          description: "Name of the image to pull. The name may include a tag or digest. This parameter may only be used when pulling an image. The pull is cancelled if the HTTP connection is closed."
+          description: "Name of the image to pull. If the name includes a tag or digest, it is ignored. Use the `tag` property to specify the desired tag or digest. This parameter may only be used when pulling an image. The pull is cancelled if the HTTP connection is closed."
```

Comment:
Ah right, @thaJeztah  is correct!

If a tag is provided here it will be used, but `tag` will take precedence if its provided.

So `fromImage=ubuntu:latest&tag=noble` would actually result in `ubuntu:noble` being considered